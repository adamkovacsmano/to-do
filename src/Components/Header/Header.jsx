import React, { Component } from "react";
import styles from "./Header.module.scss";

class Header extends Component {
  render() {
    return (
      <h1>
        <span>WHAT</span> to <span>DO???</span> when Life <span>SLAPS</span> you
        with a <span>FISH</span>?
      </h1>
    );
  }
}

export default Header;
