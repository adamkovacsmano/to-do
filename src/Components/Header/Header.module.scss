h1 {
  color: rgb(44, 84, 148);
  margin: 0px;
}

span {
  color: rgb(88, 133, 204);
}
