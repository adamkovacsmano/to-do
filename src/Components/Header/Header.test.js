import React from "react";
import Header from "./Header";
import { shallow } from "enzyme";

describe("FilterIcon Tests", () => {
  let header;
  beforeEach(() => {
    header = shallow(<Header />);
  });

  test("should match the snapshot", () => {
    expect(header).toMatchSnapshot();
  });
});
