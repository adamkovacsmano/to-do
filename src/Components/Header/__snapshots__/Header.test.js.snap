// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterIcon Tests should match the snapshot 1`] = `
<h1>
  <span>
    WHAT
  </span>
   to 
  <span>
    DO???
  </span>
   when Life 
  <span>
    SLAPS
  </span>
   you with a 
  <span>
    FISH
  </span>
  ?
</h1>
`;
